EFCore/MAUI/Readme.md

```diff
@@ -0,0 +1,873 @@
+
+        private async void OnArchive(object obj) {
+            if (!await DataStore.UserCanCreatePostAsync()) {
+                await Shell.Current.DisplayAlert("Error", "Permission denied", "OK");
+            }
+            else {
+                await DataStore.ArchivePostAsync(Post);
+            }
+        }
+
+        public Command ArchiveCommand { get; }
+    }
+   ```
+
+   _Views/ItemDetailPage.xaml_
+   ```xml
+   <ContentPage.ToolbarItems >
+       <ToolbarItem Text="Archive" Command="{Binding ArchiveCommand}"></ToolbarItem>
+   </ContentPage.ToolbarItems>
+   ```
+
+![](../../images/MAUI/Archive.png)
+
+### Add reporting functionality
+
+The [XAF Reports module](https://docs.devexpress.com/eXpressAppFramework/113591/shape-export-print-data/reports/reports-v2-module-overview?p=netframework) is a [Universal Subscription](https://www.devexpress.com/subscriptions/universal.xml) feature that you can use to easily integrate [DevExpress Reports](https://www.devexpress.com/subscriptions/reporting/) into your backend Web API service. Skip this step if you are using the Web API service as a part of the _DevExpress .NET App Security Library_ & Web API service free offer.
+
+Follow the steps bellow to create and initialize a report:
+
+1. Add the Report component using the Visual Studio New Item wizard.
+2. Drag and drop a CollectionDataSource component from the Visual Studio toolbox and change its ObjectTypeName to `WebAPI.BusinessObjects.Post`.
+3. Drag & drop all discovered fields from the Field List window onto the Report details surface.
+
+   ![](../../images/MAUI/ReportDesigner.png) 
+4. Use a predefined reports updater to initialize the report (Module.cs).
+   ```cs
+   public override IEnumerable<ModuleUpdater> GetModuleUpdaters(IObjectSpace objectSpace, Version versionFromDB) {
+        var predefinedReportsUpdater = new PredefinedReportsUpdater(Application, objectSpace, versionFromDB);
+        predefinedReportsUpdater.AddPredefinedReport<XtraReport1>("Post Report",typeof(Post));
+        return new ModuleUpdater[] { new DatabaseUpdate.Updater(objectSpace, versionFromDB),predefinedReportsUpdater };
+    }
+   ```
+See the [Create a Report in Visual Studio](https://docs.devexpress.com/XtraReports/14989/get-started-with-devexpress-reporting/create-a-report-in-visual-studio#add-a-new-report) documentation topic for more information on creating and editing reports.
+
+For more information on using predefined static report in XAF, see the [Create Predefined Static Reports](https://docs.devexpress.com/eXpressAppFramework/113645/shape-export-print-data/reports/create-predefined-static-reports) article.
+
+> Watch video: [Preview Reports as PDF in .NET MAUI Apps using Backend Web API service Endpoints with EF Core](https://www.youtube.com/watch?v=bn4iF5Gc9XY)
+
+5. Create the `GetReport` endpoint which will redirect to build-in `DownloadByName` endpoint and will return a Report for the Post with title _Post Report_.
```

Comment:
The phrase "for the Post" implies that the report will only contain information about a single Post. I don't think that's correct. 

```suggestion
5. Create a `GetReport` endpoint. It redirects to the built-in `DownloadByName` endpoint and returns a Report with title _Post Report_.
```